.circleci/config.yml

```diff
@@ -1,12 +1,13 @@
 version: 2
 jobs:
-  build:
+  test-frontend-and-server:
     docker:
-      - image: cypress/browsers:chrome67
+      #      - image: cypress/browsers:chrome67
+      - image: circleci/python:3.6.7-node-browsers
```

Comment:
nit: Does this image contain all browsers in commented line above. If so, delete comment.

Reply:
yes it does, I'll delete.